src/ProcessSlave.php

```diff
@@ -410,7 +410,7 @@ public function onRequest(ServerRequestInterface $request)
         $logTime = date('d/M/Y:H:i:s O');
 
         $catchLog = function ($e) {
-            console_log((string) $e);
+            \console_log((string) $e);
```

Comment:
I'm afraid this is the reason the tests are failing, console_log does not live in the root namespace, it's a custom function.

Reply:
Missed that. Thanks!